mycroft/skills/media/vocab/en-us/FromKeyword.voc

```diff
@@ -0,0 +1 @@
+from
```

Comment:
It was at one point, currently it isn't. It might not be necessary at all come to think of it...I used it when I want to specify a skill exclusively to handle a play request...
```py
intent = IntentBuilder('PlayFromIntent' + self.name)\
            .require('PlayKeyword')\
            .require('PlaylistKeyword')\
            .require('FromKeyword')\
            .require('NameKeyword')\
            .build()
```

but the from keyword isn't really necessary is it?

```py
intent = IntentBuilder('PlayFromIntent' + self.name)\
            .require('PlayKeyword')\
            .require('PlaylistKeyword')\
            .require('NameKeyword')\
            .build()
```
would work just as nicely right?